### PR TITLE
Jfj integration

### PR DESCRIPTION
**Pull request by kferjaoui**
Integration of JFJ controls in the JFgui

### CODE REVIEW

File: jungfrau_gui/decoder.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import zmq
+import cbor2
+from .dectris.compression import decompress
```
Comment (erikfrojdh) [inline jungfrau_gui/decoder.py:3]: Should this not be
```python
from dectris.compression import decompress
```
without the leading dot?

File: jungfrau_gui/decoder.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+
+def tag_hook(decoder, tag):
+    tag_decoder = tag_decoders.get(tag.tag)
+    return tag_decoder(tag) if tag_decoder else tag
+
+
+
+if __name__ == "__main__":
```
Comment (erikfrojdh) [inline jungfrau_gui/decoder.py:66]: If you keep this in make the endpoint configurable from the command line. But can also be removed.

**Review by erikfrojdh — COMMENTED** [review]
Looks ok and you can merge. But as discussed yesterday I would consider supporting Jungfraujoch only.

File: jungfrau_gui/decoder.py (hunk 1 above)
Comment (kferjaoui) [reply jungfrau_gui/decoder.py:3]: This works as I built the compression (.so) package from scratch in the "broken" (dev) env and put in the dectris folder... Will fix the env and make the changes as you suggest

**Pull request merged**